video_mapper.go

```diff
@@ -278,6 +279,15 @@ func (v videoMapper) mapBrightcoveVideo(brightcoveVideo map[string]interface{},
 	b := brand{
 		ID: FTBrandID,
 	}
+	longDescription, _ := get("long_description", brightcoveVideo)
```

Comment:
Might want to extract this into a function & inline the only place where it's used?
It's fine, just remember this function is quite long.